movingSquare.js

```diff
@@ -0,0 +1,93 @@
+const square = document.createElement('div')
+const h6 = document.createElement('h6')
+
+let positionX = 0
+let positionY = 0
+
+square.style.backgroundColor = 'red'
+square.style.width = '100px'
+square.style.height = '100px'
+square.style.position = 'absolute'
+square.style.left = positionX + 'px'
+square.style.top = positionY + 'px'
+square.style.zIndex = 1
+
+h6.innerHTML = 'X=' + String(positionX) + ' | ' + 'Y=' + String(positionY)
+h6.style.position = 'absolute'
+h6.style.left = 0
+h6.style.top = 0
+h6.style.margin = '4px'
+h6.style.zIndex = 2
+
+document.body.appendChild(square)
+document.body.appendChild(h6)
+
+changer = () => {
+  square.style.top = positionY + 'px'
+  square.style.left = positionX + 'px'
+  h6.innerHTML = 'X=' + String(positionX) + ' | ' + 'Y=' + String(positionY)
```

Comment:
Each function should have only 1 responsibility. This has 2. 

1. Updating square.
2. Updating stats.

Lets go with 2 functions.